client/src/components/create-collection-form/form-steps/LayerImageUpload.tsx

```diff
@@ -137,19 +143,34 @@ const LayerImageUpload = ({
           );
         })}
       </Box>
-      <Paper>
-        <Input
-          type="number"
-          value={state.generative.quantity}
-          onChange={handleQuantityChange}
-          placeholder="0"
-          label="Collection Quantity"
-          InputProps={{
-            inputProps: { min: 1, max: getMaxQuantity() },
-          }}
-          required
-        />
-      </Paper>
+      <Collapse in={getMaxQuantity() > 0}>
+        <p>
+          Maximum possible collection quantity: {getMaxQuantity()}.{" "}
+          <Collapse
+            in={
+              getMaxQuantity() !==
+              Math.ceil(SIZE_RECOMMENDATION * getMaxQuantity())
+            }>
+            <p>
+              We recommend generating up to{" "}
+              {Math.ceil(SIZE_RECOMMENDATION * getMaxQuantity())}.
+            </p>
+          </Collapse>
+        </p>
+        <Paper>
+          <Input
```

Comment:
Remove quantity input from Collapse, otherwise it won't show validation error when no images uploaded